src/llmperf/datasets/gpqa.py

```diff
@@ -62,8 +56,8 @@ def randomly_sample_gpqa_prompt(
         prompt = QUERY_TEMPLATE_MULTICHOICE.format(**choices_dict)
         prompts.append(prompt)
 
-        if get_token_length(prompt) < min_prompt_length:
-            min_prompt_length = get_token_length(prompt)
+        if get_token_len(prompt) < min_prompt_length:
+            min_prompt_length = get_token_len(prompt)
```

Comment:
https://docs.python.org/3.13/whatsnew/3.8.html#assignment-expressions

```suggestion
        if (token_len := get_token_len(prompt)) < min_prompt_length:
            min_prompt_length = token_len
```

Reply:
반영했습니다. [d42421e](https://github.com/furiosa-ai/llmperf/pull/3/commits/d42421e17d67113adf35f96becfe126a0b735153)